addons/website_sale/views/templates.xml

```diff
@@ -1502,6 +1502,8 @@
                                         </tbody>
                                     </table>
                                     <t t-call="website_sale.payment_confirmation_status" />
+                                    <t t-set='connect' t-value="request.env['ir.config_parameter'].sudo().get_param('auth_signup.allow_uninvited') == 'True' and ('signup', 'Sign Up') or ('login', 'Log In')"/>
+                                    <t t-if="connect[0] == 'signup'"><a t-attf-href='/web/{{connect1[0]}}?redirect=/shop/checkout' style="margin-top: -11px">Sign Up Now to Follow Your Order</a></t>
```

Comment:
branch is red, and nothing is correct !!!!!

`if connect[0] == 'signup'`
 => 
`if request.env['ir.config_parameter'].sudo().get_param('auth_signup.allow_uninvited') == 'True'`

What happen if already logged in ? 
=> `and request.website.user == request.env.user`

{{connect1[0]}} => Probably not tested before to submit ! ...  maybe {{connect[1]}} ?

redirect=/shop/checkout => Why return on checkout one the order finished, no sense !

Signup now to follow your order => purpose is to signup with the partner that has just been created.
If you send the customer to /web/signup, is it just useless, he will create a second partner and don't see the order...
It was all the purpose of this task.
=>
`order.partner_id.signup_prepare() and order.partner_id.signup_url`


In summary:

```diff
+    <t t-if="request.env['ir.config_parameter'].sudo().get_param('auth_signup.allow_uninvited') == 'True' and request.website.is_public_user()">
+        <t t-if="request.env['ir.config_parameter'].sudo().get_param('auth_signup.allow_uninvited') == 'True' and request.website.is_public_user()">
+         <p class='alert alert-info mt16'>
+            <a t-att-href='order.partner_id.signup_prepare() and order.partner_id.signup_url' class='btn btn-primary'>Sign Up</a>
+             Now to Follow Your Order
+          </p>
+       </t>
+    </t>


```



**Thanks to proposing your PR and mark as done once it has been tested and match the purpose of the task!
If you don't want to do this one, there are many other, but just don't mark this as done with random code and let the bug to somebody that is interested to fix it!**